imports/plugins/included/search-mongo/server/methods/searchcollections.js

```diff
@@ -242,7 +242,7 @@ export function buildOrderSearchRecord(orderId) {
   orderSearch.variants.title = order.items.map((item) => item.variants && item.variants.title);
   orderSearch.variants.optionTitle = order.items.map((item) => item.variants && item.variants.optionTitle);
 
-  OrderSearch.insert(orderSearch);
+  OrderSearch.upsert(orderId, { $set: { ...orderSearch } });
```

Comment:
I think additionally we should surround this in a `try/catch`. There's no reason that anything that happens here should block settings the shipping status or anything else. So we should just catch and log any errors.

Reply:
OK doing that.